crates/light-client-prover/src/da_block_handler.rs

```diff
@@ -87,7 +90,7 @@ where
         );
         tokio::pin!(l1_sync_worker);
 
-        let mut interval = tokio::time::interval(Duration::from_secs(1));
+        let mut interval = tokio::time::interval(Duration::from_secs(2));
```

Comment:
Why did this value increase?

Reply:
Esad initialized it with 1 sec in his branch, but I thought a higher interval would be more appropriate due to bitcoin producing blocks in between 1-10 minutes. One second seemed unnecessarily a bit too frequent to me.